### DISCUSSION TITLE
Should a more secure cluster be achieved by restricting listening IP?

### DISCUSSION THREAD

**Discussion by 2233466866** [rabbitmq/rabbitmq-server#8907] [Other] [answered]
### Is your feature request related to a problem? Please describe.

![issue](https://github.com/rabbitmq/rabbitmq-server/assets/17674930/b5a8c43d-9ad3-4122-90e0-fcda5c05e7d7)
![issue1](https://github.com/rabbitmq/rabbitmq-server/assets/17674930/63dcb10b-db7d-4f6f-8f46-f3e3a2476ce1)
![issue2](https://github.com/rabbitmq/rabbitmq-server/assets/17674930/5fe12977-5073-4581-a748-2cc963a980c8)

### Describe the solution you'd like

Just like the following configuration:
```
[
  {
    kernel,
    [
      %% {inet_dist_listen_ip,"192.168.1.123"},
      {inet_dist_listen_min,25678},
      {inet_dist_listen_max,25678}
    ]
  }
].
```

**Comment by mkuratczyk** [comment, marked as answer]
The address on which EPMD listens is already configurable: https://www.rabbitmq.com/networking.html#epmd-interface

While restricting the interfaces on which EPMD is listening is reasonable, assigning non-standard ports to everything is just pointless and confusing security-by-obscurity if you ask me. But it's up to you.

**Comment by michaelklishin** [comment]
One more thing that should be mentioned is that epmd communication protocol is very limited and some operations am an inly be performed by a client (such as a RabbitMQ node) that runs on the same host.

So the attack surface is tiny, and all communication that happens after peer host and port discovery via epmd, still uses a shared secret authentication mechanism.
